nvme.c

```diff
@@ -4626,15 +4630,21 @@ static int get_feature_ids(struct nvme_dev *dev, struct feat_cfg cfg)
 	for (i = cfg.feature_id; i < feat_max; i++, feat_num++) {
 		cfg.feature_id = i;
 		err = get_feature_id_changed(dev, cfg, changed);
-		if (err && !nvme_status_equals(filter_out_flags(err),
-					       NVME_STATUS_TYPE_NVME,
-					       NVME_SC_INVALID_FIELD))
+		if (!err)
+			continue;
```

Comment:
`get_features_id_changed()` calls `get_feature_id()` which returns the the status from `nvme_get_features()`. This would be 0/1 and errno. 

(need to look a bit closer tomorrow)

Reply:
The status is checked by `get_feature_id_print()` but seems to be `0`: success, `> 0`:  error status and `-errno`. Also it is checked as same by other commands.